lib/src/main/kotlin/com/swmansion/starknet/crypto/FeeUtils.kt

```diff
@@ -0,0 +1,15 @@
+package com.swmansion.starknet.crypto
+
+import com.swmansion.starknet.data.types.Felt
+import java.math.BigInteger
+import kotlin.math.roundToInt
+
+internal object FeeUtils {
+    @JvmStatic
+    fun estimatedFeeToMaxFee(fee: Felt, overhead: Double = 0.5): Felt {
```

Comment:
I wonder if default overhead of `0.5` is a good choice. In starknet.py we're using 0.1 i believe.